### PR TITLE
Backport webscale tests

### PR DESCRIPTION
**Pull request by achilleasa**
## Description of change

This PR backports the webscale tests from develop into 2.5

### CODE REVIEW

**Comment by achilleasa** [comment]
!!build!!

**Comment by achilleasa** [comment]
!!build!!

File: acceptancetests/deploy_stack.py [hunk 1]
```diff
@@ -73,7 +74,7 @@
 log = logging.getLogger(__name__)
 
 
-LXD_PROFILE = """
+CAAS_LXD_PROFILE = """
```
Comment (mitechie) [inline acceptancetests/deploy_stack.py]: CAAS? this looks odd.

File: acceptancetests/deploy_stack.py [hunk 2]
```diff
@@ -137,24 +147,75 @@ def deploy_dummy_stack(client, charm_series, use_charmstore=False):
...
+    client = _deploy_stack(
+        path, client,
+        timeout=timeout,
+        charm=charm,
+        lxd_profile=IAAS_LXD_PROFILE,
+    )
+    return IaasClient(client)
+
+
+def deploy_caas_stack(path, client, timeout=3600, charm=False):
```
Comment (mitechie) [inline acceptancetests/deploy_stack.py]: I see, the CAAS name I think is really off here. Deploying a k8s cluster isn't really CAAS as that's related to the k8s charms work that's gone on. If it's what's there though this is an ok backport.

Comment (achilleasa) [reply acceptancetests/deploy_stack.py]: I just copied the files over from develop to make sure there is no conflict when we merge 2.5 -> develop.

**Comment by achilleasa** [comment]
$$merge$$

**Pull request merged**